kakao/src/main/kotlin/com/agoda/kakao/delegates/ViewInteractionDelegate.kt

```diff
@@ -0,0 +1,47 @@
+package com.agoda.kakao.delegates
+
+import android.support.test.espresso.*
+import android.view.View
+import com.agoda.kakao.configurator.KakaoConfigurator
+import org.hamcrest.Matcher
+
+/**
+ * An interface duplicating the used in Kakao part of view interaction interface and providing original Espresso *Interactions.
+ * If you want to extend the base view interaction behavior, you should implement this interface, and specify it's
+ * factory in [com.kaspersky.klkakao.configurator.KakaoConfigurator]
```

Comment:
`factory in [com.kaspersky.klkakao.configurator.KakaoConfigurator]` not sure that links to external libraries is permitted

Reply:
As I mentioned above in my first message at this PR I would update docs later